addon/components/o-s-s/number-input.stories.js

```diff
@@ -0,0 +1,75 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::NumberInput',
+  component: 'number-input',
+  argTypes: {
+    value: {
+      description: '[OPTIONAL] The starting value the input will take. Defaults to 0.',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'number' }
+    },
+    min: {
+      description: '[OPTIONAL] The minimum value the number can be.',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'number' }
+    },
+    max: {
+      description: '[OPTIONAL] The maximum value the number can be.',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'number' }
+    },
+    step: {
+      description: '[OPTIONAL] The increase & decrease value of each button press. Defaults to 1.',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'number' }
+    },
+    onChange: {
+      description: 'A callback that sends back the new value of the input'
+    }
+  },
+  parameters: {
+    docs: {
+      description: {
+        component:
+          '[OPTIONAL] A number input wrapper with decrease & increase buttons. (Up & Right arrow increase, Down & Left arrow decrease)'
+      }
```

Comment:
```suggestion
          'A number input wrapper with decrease & increase buttons. (Up & Right arrow increase, Down & Left arrow decrease)'
      }
```
Not necessary in components description